packages/terra-site/src/cerner-mock-theme.scss

```diff
@@ -47,10 +48,15 @@
   --terra-form-checkbox-checked-border-style: solid;
   --terra-form-checkbox-checked-border-width: 1px;
   --terra-form-checkbox-symbol-color: rgb(255, 255, 255);
-  --terra-form-checkbox-symbol: '\2713';
+  --terra-form-checkbox-symbol: url('data:image/svg+xml;utf8,<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path d="M18.4 40.9L0 22.5l4.2-4.2 14.2 14.2L43.8 7.1l4.2 4.2z"/></svg>');
```

Comment:
@kschuste added a mixin to help encode SVGs in his PR: https://github.com/cerner/terra-core/blob/ee14d191262dcdfacf2e04ad76d84ad2a28c84a6/packages/terra-mixins/src/Mixins.scss

We should copy that mixin over to your branch and use that for this SVG. Be sure to test this in IE too, we ran into some issues with the utf8 charset that was used for the svg. Using `data:image/svg+xml;charset=US-ASCII,#{url-encode($string)}` which the mixin will provide seemed to work better in IE.

Reply:
Mixin added [here](https://github.com/cerner/terra-core/pull/1037/commits/ad2cba45da51dbc5d6e294681b2b326653159b0c#diff-165d975ef87fbc0f0b863c7984cb1834R575). 

There was an IE 10 issue where the label text was being pushed under the checkbox for some custom themes. Fixed [here](https://github.com/cerner/terra-core/pull/1037/commits/ad2cba45da51dbc5d6e294681b2b326653159b0c#diff-222d72b921649ce14186f14e6910d91bR34).